# bug_index_error.py
my_list = [1, 2, 3]
print(my_list[5])